### DISCUSSION TITLE
Swap around `gz` and `gs`?

### DISCUSSION THREAD

**Discussion by briandipalma** [LazyVim/LazyVim#201] [General]
Thanks for the amazing plugin. I imagine most people use surround functionality more than searching in other windows and it would fit with `gc` a bit better? Maybe make `gs` `gS`?

**Comment by dpetka2001** [comment]
You can change the keys to whatever you want. Let the default configuration be what it is and whoever wants can change it according 
to their preferences. See this [issue](https://github.com/LazyVim/LazyVim/issues/172) in which the dev pushed some changes to make it easier to change mappings for `mini.surround`. You should only need to change the appropriate `opts.mappings` to do whatever you like.

**Comment by folke** [comment]
As @dpetka2001 mentions, you can just change the options for your config. I'd rather keep the current keymaps as the default

**Comment by briandipalma** [comment]
I know I can change the mappings. I thought I'd mention this as a possibility for a 2.0 release. I've no idea if there is a list somewhere of possible breaking changes but I thought I'd mention this as something that could be discussed for such a release.

It's a pity I can't mark this discussion as resolved or closed but it should be considered so.
